libs/layout/src/lib/abstracts/display-content/display-content.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Directive, Input } from '@angular/core';
+
+/**
+ * An abstract class to use when implementing custom fallbacks for the NgxDisplayContentDirective
+ */
+@Directive()
+export abstract class NgxDisplayContentComponent {
+	/**
+	 * An optional data input to provide data to the fallback components
+	 */
+	@Input() data: any;
```

Comment:
I would change this to a `never` or `unknown`, or make a mention in the comments that this is an intended `any`.